src/pages/workspace/WorkspacesListPage.js

```diff
@@ -0,0 +1,214 @@
+import ScreenWrapper from '../../components/ScreenWrapper';
+import ROUTES from '../../ROUTES';
+import ONYXKEYS from '../../ONYXKEYS';
+import CONST from '../../CONST';
+import styles from '../../styles/styles';
+import compose from '../../libs/compose';
+import OfflineWithFeedback from '../../components/OfflineWithFeedback';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import * as Expensicons from '../../components/Icon/Expensicons';
+import themeColors from '../../styles/themes/default';
+import * as PolicyUtils from '../../libs/PolicyUtils';
+import MenuItem from '../../components/MenuItem';
+import * as Policy from '../../libs/actions/Policy';
+import policyMemberPropType from '../policyMemberPropType';
+import Permissions from '../../libs/Permissions';
+import Button from '../../components/Button';
+import FixedFooter from '../../components/FixedFooter';
+import BlockingView from '../../components/BlockingViews/BlockingView';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** The list of this user's policies */
+    policies: PropTypes.objectOf(PropTypes.shape({
+        /** The ID of the policy */
+        ID: PropTypes.string,
+
+        /** The name of the policy */
+        name: PropTypes.string,
+
+        /** The type of the policy */
+        type: PropTypes.string,
+
+        /** The user's role in the policy */
+        role: PropTypes.string,
+
+        /** The current action that is waiting to happen on the policy */
+        pendingAction: PropTypes.oneOf(_.values(CONST.RED_BRICK_ROAD_PENDING_ACTION)),
+    })),
+
+    /** List of policy members */
+    policyMembers: PropTypes.objectOf(policyMemberPropType),
+
+    /** The user's wallet account */
+    userWallet: PropTypes.shape({
+        /** The user's current wallet balance */
+        currentBalance: PropTypes.number,
+    }),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    policies: {},
+    policyMembers: {},
+    userWallet: {
+        currentBalance: 0,
+    },
+};
+
+/**
+ * Dismisses the errors on one item
+ *
+ * @param {string} policyID
+ * @param {string} pendingAction
+ */
+function dismissWorkspaceError(policyID, pendingAction) {
+    if (pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
+        Policy.clearDeleteWorkspaceError(policyID);
+        return;
+    }
+    throw new Error('Not implemented');
+}
+
+class WorkspacesListPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.getWalletBalance = this.getWalletBalance.bind(this);
+        this.getWorkspaces = this.getWorkspaces.bind(this);
+        this.getMenuItem = this.getMenuItem.bind(this);
+    }
+
+    /**
+     * @param {Boolean} isPaymentItem whether the item being rendered is the payments menu item
+     * @returns {Number} the user wallet balance
+     */
+    getWalletBalance(isPaymentItem) {
+        return (isPaymentItem && Permissions.canUseWallet(this.props.betas))
```

Comment:
Not subscribed to ONYXKEYS.BETAS to read betas.

Reply:
Updated